R/FilteredData-utils.R

```diff
@@ -60,12 +60,10 @@ init_filtered_data.TealData <- function(x, # nolint
 
 #' @keywords internal
 #' @export
-init_filtered_data.default <- function(x, join_keys = teal.data::join_keys(), code = NULL, check = FALSE) { # nolint
+init_filtered_data.default <- function(x, join_keys = teal.data::join_keys(), code, check) { # nolint
```

Comment:
```suggestion
init_filtered_data.default <- function(x, join_keys = teal.data::join_keys(), code, check) { # nolint
  lifecycle::deprecate_stop(
    "0.4.1", 
    "init_filtered_data(check = 'No longer supported')"
  )
  lifecycle::deprecate_stop(
    "0.4.1", 
    "init_filtered_data(code = 'No longer supported')"
  )
```

Reply:
This raises errors on all calls, even when the arguments are not passed. I suppose it is because the `NULL` defaults in the generic.

Reply:
okey, let's do this on a feature branch or don't do this at all